collect_app/src/main/res/values/attrs.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <attr name="icon" format="reference" />
```

Comment:
What's this for?

Reply:
It's used in two places and if it's not extracted like that Android studio throws `Found item Attr/icon more than one time` see: https://stackoverflow.com/questions/4434327/same-named-attributes-in-attrs-xml-for-custom-view or https://medium.com/@bruno.lima/android-custom-views-same-attribute-on-multiple-custom-views-78e9de1ce6c8